scripts/aqm/stats/exevs_aqm_grid2obs_stats.sh

```diff
@@ -64,8 +61,14 @@ obs_hourly_found=0
 if [ -s ${check_file} ]; then
     obs_hourly_found=1
 else
-    echo "Can not find pre-processed obs hourly input ${check_file}"
-    ## add email function here
+  echo "Can not find pre-processed obs hourly input ${check_file}"
+  if [ $SENDMAIL = "YES" ]; then 
+    export subject=AQM Hourly Observed Missing for EVS ${COMPONENT}"
```

Comment:
Add an opening ". The line should be: `export subject=AQM Hourly Observed Missing for EVS ${COMPONENT}"`

Reply:
I wonder if lack of this " is what caused the errant email to be sent to Ho-Chun.

Reply:
@PerryShafran-NOAA , perhaps! It made the log file all wonky. Like the log trace did not match the script at all. I was finally able to track it down in the code itself. Like, you know how the code can change color or highlight open and closed brackets, etc ...? The coloring was funny in the SENDMAIL test block and I traced it to a missing ".

Reply:
Oh, no, it was probably the typo you had mentioned below.

Reply:
This error is fixed.